data/themes/darktable.css

```diff
@@ -2757,3 +2757,10 @@ spinbutton>button
 {
     font-weight: bold;
 }
+
+popover separator
+{
+   margin: 2px 0 2px;
+   min-height: 1px;
+   background-color: @bauhaus_fg;
```

Comment:
Just remove all those lines you add in css and simply add 'popover separator' to 'menu separator in line 991, like this:

```
menu separator,
popover separator
{
  margin: 0.6em 2em;
  padding-bottom: 1px;
  background-color: @selected_bg_color;
}
```

It will be all good and consistent to other similar separators.

Reply:
Thanks a lot ! Now changed and looks better indeed.